script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -1025,12 +1025,13 @@ If I do not burn it to the ground, it would go against everything that I believe
 [sha:1c569a09df5b70346eb15174ccaa0b9e0ed30421]{
 -- Page 870, Offset 7977.
 -- 真祖の姫。その心臓を、おれに<呉|よ>こせ”」
-Princess of the True Ancestors. That heart of yours, shall be mine."
+Princess of the True Ancestors... That heart of yours shall be mine!"
 }
-[sha:99b90bc4120fe60c2ec33be3ba4a07d1a087fd89]{
+[offset:7978]{
 -- Page 871, Offset 7978.
 -- 「……！」
-"!"
+// Not even voiced
+This is bad.
```

Comment:
interesting

Reply:
I mean... what can you do when presented with "!"